### ISSUE TITLE
NullpointerException: MagicentoPhpCompletionContributor.getAutocompleteForClassExtends

### ISSUE THREAD

**Issue by Schrank** [enriquepiatti/Magicento#28]
I get a NullPointerException with a brand new controller file calles AskLanguageController.php. I can't send you the first name of the controller, but after writing:

`class BlubController extends <Ctrl> + <Space>` it is  still reporducable

Thanks for fixing.

```
null
java.lang.NullPointerException
    at com.magicento.helpers.MagentoParser.getClassType(MagentoParser.java:426)
    at com.magicento.helpers.MagentoParser.isBlock(MagentoParser.java:460)
    at com.magicento.extensions.MagicentoPhpCompletionContributor.getAutocompleteForClassExtends(MagicentoPhpCompletionContributor.java:542)
    at com.magicento.extensions.MagicentoPhpCompletionContributor$1.addCompletions(MagicentoPhpCompletionContributor.java:96)
    at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:36)
    at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:156)
    at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:83)
    at com.intellij.codeInsight.completion.CompletionService.performCompletion(CompletionService.java:114)
    at com.intellij.codeInsight.completion.CodeCompletionHandlerBase$2$1$1.run(CodeCompletionHandlerBase.java:357)
    at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:918)
    at com.intellij.codeInsight.completion.CodeCompletionHandlerBase$2$1.run(CodeCompletionHandlerBase.java:348)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl$2.run(ProgressManagerImpl.java:181)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:221)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcess(ProgressManagerImpl.java:172)
    at com.intellij.codeInsight.completion.CodeCompletionHandlerBase$2.run(CodeCompletionHandlerBase.java:344)
    at com.intellij.openapi.application.impl.ApplicationImpl$6.run(ApplicationImpl.java:457)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
    at java.util.concurrent.FutureTask.run(FutureTask.java:138)
    at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
    at java.lang.Thread.run(Thread.java:680)
    at com.intellij.openapi.application.impl.ApplicationImpl$1$1.run(ApplicationImpl.java:148)
```

**Comment by enriquepiatti** [comment]
Thanks Schrank, is fixed now, it will be available with the next release (I want to add some new features before releasing another version).

**Comment by enriquepiatti** [comment]
Fixed in v1.3.0
